From ad8e2cbf9ca9c1c50deec33f9ade5d892d2be9bb Mon Sep 17 00:00:00 2001
From: Kyle McMartin <kmcmarti@redhat.com>
Date: Fri, 12 Sep 2014 17:46:49 -0400
Subject: [kernel] redhat: bump RHEL_MINOR to 1

Message-id: <20140912174649.GV29420@redacted.bos.redhat.com>
Patchwork-id: 94057
O-Subject: [RHEL7.1 PATCH 5/6 v4] redhat: bump RHEL_MINOR to 1
Bugzilla: 1108378
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Johannes Weiner <jweiner@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

We're going to need to identify modules before and after the 16K stacks
change. As a result, we need to bump RHEL_MINOR at the same time as this
goes in, otherwise massive weirdness might occur.

Signed-off-by: Kyle McMartin <kmcmarti@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index aa8e56b..1ae9eeb 100644
--- a/Makefile
+++ b/Makefile
@@ -4,7 +4,7 @@ SUBLEVEL = 0
 EXTRAVERSION =
 NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
-RHEL_MINOR = 0
+RHEL_MINOR = 1
 RHEL_RELEASE = 159
 
 # *DOCUMENTATION*
-- 
1.7.1